### PR TITLE
Resolve homework Lesson 1

### PR DESCRIPTION
**Pull request by codeeMadness**
Hi Quang,

Please help to review some my homework which are resolved so far. There is still one quiz left, I'm still working on it.

Regards,
Hang Nguyen

### CODE REVIEW

File: Homework/Hang_Nguyen/lesson_1/sort-array-by-parity.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+class Solution:
+    def sortArrayByParity(self, nums):
+        last = len(nums) - 1
+        begin = 0
+        while begin < last: 
+            if nums[begin] % 2 != 0: 
+                temp = nums[begin]
```
Comment (quanghoang33) [inline Homework/Hang_Nguyen/lesson_1/sort-array-by-parity.py]: iirc, swap in python can be done by a,b = b,a

**Review by quanghoang33 — CHANGES_REQUESTED** [review]

**Comment by codeeMadness** [comment]
Hi Quang,

I have resolved all quiz. Please help to take a look and leave review if any.

Many thanks,
Hang Nguyen

File: Homework/Hang_Nguyen/lesson_1/range-sum-query-immutable.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    def __init__(self, nums):
+        self.sumLst = []
+        s = 0
+        for num in nums:
+            s += num
+            self.sumLst.append(s)
+
+    
+    def sumRange(self, left, right):
+        if left > 0 and right > 0:
```
Comment (quanghoang33) [inline Homework/Hang_Nguyen/lesson_1/range-sum-query-immutable.py]: do we need `right > 0` ?

**Review by quanghoang33 — CHANGES_REQUESTED** [review]

File: Homework/Hang_Nguyen/lesson_1/sort-array-by-parity.py (hunk 1 above)
Comment (codeeMadness) [reply Homework/Hang_Nguyen/lesson_1/sort-array-by-parity.py]: Hi thay Quang,

I have updated as your feedback.

Many thanks,
Hang Nguyen

File: Homework/Hang_Nguyen/lesson_1/range-sum-query-immutable.py (hunk 2 above)
Comment (codeeMadness) [reply Homework/Hang_Nguyen/lesson_1/range-sum-query-immutable.py]: Hi thay Quang,

I have removed redundant condition. You can help to take a look.

Many thanks,
Hang Nguyen

**Pull request merged**